docs/config/index.md

```diff
@@ -237,6 +237,26 @@ Custom reporters for output. Reporters can be [a Reporter instance](https://gith
   - `'json'` -  give a simple JSON summary
   - path of a custom reporter (e.g. `'./path/to/reporter.ts'`, `'@scope/reporter'`)
 
+### outputTruncateLength
+
+- **Type:** `number`
+- **Default:** `80`
+
+Truncate output diff lines up to `80` number of characters. You may wish to tune this,
+depending on you terminal window width.
+
+To provide object via CLI command, use the following syntax: `--outputTruncateLength=100`.
+
+### outputDiffLines
+
+- **Type:** `number`
+- **Default:** `15`
+
+Limit number of output diff lines up to `15`. You may wish to tune this, to see output
+for larger objects/arrays.
+
+To provide object via CLI command, use the following syntax: `--outputDiffLines=100`.
```

Comment:
what object? 😄 

I think you've taken it from `outputFile` command, where it's possible to provide object via CLI, but here it' just a number. I would remove this line and on `:248`